docs/guide/000-introduction.adoc

```diff
@@ -1,14 +1,15 @@
 
 This part of the Guide is intended for metadata authors and product or infrastructure specialists who create WCMP 1.3 metadata records for making their datasets discoverable within the WIS catalogue(s). It will also assist those who wish to create high-quality WIS metadata records for data that will be ingested and distributed by a GISC.
 
 The WCMP templates and relevant documentation listed below should be used together with the guidance information provided in this part of the Guide, which also contains a set of recommendations to be followed in order to provide the right level and granularity of product information in the WCMP metadata records:
 
-- WCMP 1.3 Template XML records: Template records containing placeholders (to be replaced with information related to the specific product described by the WIS discovery metadata record) are available from http://wis.wmo.int/MD-Templates. A valid example of an XML record, with field content (to be replaced) is also available there;
-- WIS Wiki Page on WIS discovery metadata: http://wis.wmo.int/MD_Index;
-- WCMP documentation: Part 1: http://wis.wmo.int/WCMPpart1; Part 2: http://wis.wmo.int/WCMPPart2;
-- Additional guidance WCMP documentation: http://wis.wmo.int/MD_Index or http://wis.wmo.int/WIS-Manual (for a summary of changes);
-- Additional examples of WCMP metadata (for particular product types): https://wis.wmo.int/MD-Examples.
-
+- WCMP 1.3 Template XML records: Template records containing placeholders (to be replaced with information related to the specific product described by the WIS discovery metadata record) are available from https://old.wmo.int/wiswiki/tiki-index.php%3Fpage=MDG_Guide.html. A valid example of an XML record, with field content (to be replaced) is also available there;
+- WIS Wiki Page on WIS discovery metadata: https://community.wmo.int/activity-areas/wis/wcmp;
+- WCMP documentation: 
+* Part 1 - Conformance Requirements provides the basic definition of the Core Profile : http://wis.wmo.int/WCMPpart1; 
+* Part 2 - Abstract Test Suite, Data Dictionary and Code Lists describes the constraints on how metadata may be written and defines the definitions of terms used within the metadata : http://wis.wmo.int/WCMPPart2;
+- Additional guidance WCMP documentation: https://old.wmo.int/wiswiki/tiki-index.php%3Fpage=IndexForMetadataGuidance.html ;
+- Additional examples of WCMP metadata (for particular product types): https://old.wmo.int/wiswiki/tiki-index.php%3Fpage=MDG_EG.html
```

Comment:
@amilan17 please check links - thx!

Reply:
I think we should remove all of these URLs, except for the link to the WCMP homepage on the Community site.

Reply:
I will work on the draft for this section.